Dockerfile

```diff
@@ -4,13 +4,14 @@ ARG BASE_IMAGE=docker.io/library/ubuntu:21.10
 # Initial stage which pulls prepares build dependencies and CLI tooling we need for our final image
 # Also used as the image in CI jobs so needs all dependencies
 ####################################################################################################
-FROM docker.io/library/golang:1.18 as builder
+FROM docker.io/library/golang:1.18 AS builder
 
 RUN echo 'deb http://deb.debian.org/debian buster-backports main' >> /etc/apt/sources.list
 
-RUN apt-get update && apt-get install -y \
+RUN apt-get update && apt-get install --no-install-recommends -y \
     openssh-server \
     nginx \
+    unzip \
```

Comment:
Why the extra dependency?

Reply:
may be bcoz ADD supports  auto extract of files and COPY doesn't

Reply:
i have run it locally and without unzip we got

#20 21.71 + unzip /tmp/dl/awscli-linux-amd64-v2.4.6.zip -d /tmp/awscliv2
#20 21.71 ./installers/install-awscli-linux.sh: line 19: unzip: command not found

Reply:
As i understand unzip was installed as part of apt-get install without no-install-recommends, and once we added it unzip stopped to be part of installation that cause problems above

Reply:
Sorry for 0 response till now, family issues. Yes as @pasha-codefresh stated above with no install recommends argument I couldn't unzip that's why I have added it. ADD got nothing to do with it afaik